internal/support/logging/server_test.go

```diff
@@ -114,83 +126,103 @@ func TestGetLogs(t *testing.T) {
 		name     string
 		url      string
 		status   int
+		maxLimit int
 		criteria matchCriteria
 	}{
 		{"withoutParams",
 			"",
 			http.StatusOK,
+			100,
 			matchCriteria{}},
 		{"limit",
 			"1000",
 			http.StatusOK,
+			100,
 			matchCriteria{Limit: 1000}},
 		{"limitToLow",
 			"1",
 			http.StatusRequestEntityTooLarge,
+			1,
```

Comment:
Some tests should be added about limits. MaxReadLimit being lower than the limit in url and viceversa.

Reply:
See above. If we take your approach, we don't need these tests.